### ISSUE TITLE
scrapy runspider in case of several spiders in a file

### ISSUE THREAD

**Issue by lopuhin** [scrapy/scrapy#2608]
[Docs](https://doc.scrapy.org/en/latest/topics/commands.html?highlight=runspider#std:command-runspider) say: "run a spider self-contained in a Python file, without having to create a project."

In case there are multiple spiders in a file, one of them will be selected and run, but spider selection is not deterministic.

Although it's technically not backwards compatible, maybe the best course of action would be to raise an error in this case and do not run any spiders?

**Comment by kmike** [comment]
I think we should at least show a warning; this issue is somewhat similar to https://github.com/scrapy/scrapy/pull/2612.

**Comment by lopuhin** [comment]
Yes, a warning works great - it won't break anything and will still help with debugging. 👍

**Comment by redapple** [comment]
what about having a `--spider <spidername>` option like for `scrapy parse`?
